ui-e2e/test.js

```diff
@@ -131,6 +131,7 @@ test.describe('e2e-test for bridge.poa, version 1.5.0', async function() {
 
   test.describe('ERC TO ERC', async () => {
     test.it('User is able to open main page of bridge-ui ', async () => {
+      await foreignAccount.setMetaMaskNetwork()
```

Comment:
It was missing when running only ERC-TO-ERC, without running NATIVE-TO-ERC portion before that.

Reply:
Should we have the same in `erc-to-native` section bellow?

Reply:
In `erc-to-native` section it already exists: https://github.com/poanetwork/tokenbridge/blob/9172ad98101d72b23d3ded9a8dbe0bcb119976a5/ui-e2e/test.js#L236

Reply:
I think that this case illustrates that there is some room for improvement in the `ui-e2e` tests. The preconditions for tests like `await foreignAccount.setMetaMaskNetwork() ` are unclearly hidden in other tests, and the tests rely on sequential execution - I think we could create an issue to refactor the `ui-e2e` a little bit, using `describe` and `before` to make them more clear.

Reply:
yes, please create the issue with detailed description what is expected to do under it.

Reply:
https://github.com/poanetwork/tokenbridge/issues/188